awx_collection/plugins/modules/tower_receive.py

```diff
@@ -162,7 +162,7 @@ def main():
 
     module = TowerModule(argument_spec=argument_spec, supports_check_mode=False)
 
-    module.deprecate(msg="This module is being moved to a different collection. Instead of awx.awx it will be migrated into awx.tower_cli", version="3.7")
+    module.deprecate(msg="This module is deprecated and will be replaced by the AWX CLI export command.", version="3.7")
```

Comment:
@jbradberry did I get this right?

Reply:
I think it might be the other way around?

Reply:
Alan is correct, receive is for local to get a copy of data from an instance, send is for local to push data into an instance.